pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -74,7 +75,10 @@ func totalImageSize(nodeInfo *schedulercache.NodeInfo, containers []v1.Container
 
 	imageSizes := nodeInfo.ImageSizes()
 	for _, container := range containers {
-		if size, ok := imageSizes[container.Image]; ok {
+		name, err := parsers.GetFullImageName(container.Image)
+		if err != nil {
+			glog.V(1).Infof("Skip misnamed image: %s", container.Image)
```

Comment:
This error is reported when parse image name, which is unexpected (e.g. illegal string), we should warning it out I think.

Reply:
Reasonable - done!